examples/nitclk/synchronize-tclk.py

```diff
@@ -0,0 +1,167 @@
+# Demonstrates how to configure multiple NI Arbitrary Waveform Generators to generate synchronized waveforms with TClk
+# 
+# The gRPC API is built from the C API. NI-FGEN documentation is installed with the driver at:
+# C:\Program Files (x86)\IVI Foundation\IVI\Drivers\niFgen\documentation\English\SigGenHelp.chm
+#
+# The gRPC API is built from the C API. NI-TClk documentation is installed with the driver at:
```

Comment:
You don't need to repeat `The gRPC API is built from the C API.` two times. You can remove this from TClk documentation here.